.github/workflows/ci.yml

```diff
@@ -0,0 +1,84 @@
+---
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Python CI
+
+# yamllint disable-line rule:truthy
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+
+jobs:
+  CI-Python:
+    runs-on: ubuntu-latest
+    env:
+      PYTHONDEVMODE: 1
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.11", "3.12", "3.13"]
+        # Empty is latest, head is latest from GitHub
+        pdm-version: [""]
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: pdm-project/setup-pdm@main
+        name: Setup PDM
+        with:
+          cache: true
+          python-version: ${{ matrix.python-version }}             # Version range or exact version of a Python version to use, the same as actions/setup-python
+          architecture: x64                # The target architecture (x86, x64) of the Python interpreter. the same as actions/setup-python
+          version: ${{ matrix.pdm-version }}                   # The version of PDM to install. Leave it as empty to use the latest version from PyPI, or 'head' to use the latest version from GitHub
+          allow-python-prereleases: true  # Allow prerelease versions of Python to be installed. For example if only 3.12-dev is available, 3.12 will fall back to 3.12-dev
```

Comment:
Maybe false to only drive efforts to true releases ?

Reply:
As you want, I saw that the Docker image was using 3.13, so I thought it was a good idea!

Building and testing against future versions of Python should not be a huge issue, as there are not many dependencies and it's a pure Python package.